src/status_im/ui/screens/main_tabs/views.cljs

```diff
@@ -65,7 +65,8 @@
      ^{:key view-id} [tab view-id content (= view-id current-view-id) accessibility-label count-subscription])])
 
 (views/defview main-tabs []
-  (views/letsubs [view-id [:get :view-id]]
+  (views/letsubs [view-id          [:get :view-id]
+                  tab-bar-visible? [:get :tab-bar-visible?]]
```

Comment:
We are moving away from the pattern of using:

`[:get :something]`

changing them to:

`[:get-something]`

Could you please create a new subscription for this event? Similarly for `[:set :something]`

Reply:
We could use these events:

`hide-tab-bar`
`show-tab-bar`

and the subscription:

`:tab-bar-visible?`